### ISSUE TITLE
Add binary search

### ISSUE THREAD

**Issue by KartikSaxena19** [TheAlgorithms/Python#12127]
### What would you like to share?

In this i want to add binary search ques answer  please add the label of gssoc ext and hacktober fest

**Comment by fantazio1** [comment]
Do you mean https://github.com/TheAlgorithms/Python/blob/master/sorts/bubble_sort.py or something else?

**Comment by zhudingsuifeng** [comment]
您好，邮件已经收到，谢谢

**Comment by KartikSaxena19** [comment]
@fantazio1  no.... I want to create dsa ques using bubble sort

**Comment by PriyanshuPilaniwala032** [comment]
So, you want a proper question, like we see on leetcode on bubble sort?

**Comment by KartikSaxena19** [comment]
Yes Something like that...as it improve the learner skills that in which type of question what approach should he/she will make @PriyanshuPilaniwala032

**Comment by KartikSaxena19** [comment]
Should i work on this ?

**Comment by KartikSaxena19** [comment]
@fantazio1

**Comment by zendegiplus** [comment]
[مصطفی ضابط](https://zendegiplus1.ir/teacher/%d9%85%d8%b5%d8%b7%d9%81%db%8c-%d8%b6%d8%a7%d8%a8%d8%b7/)

**Comment by poorva-gupta** [comment]
hi, can you please assign this issue to me.
